src/find/matchers/mod.rs

```diff
@@ -252,6 +256,53 @@ fn convert_arg_to_comparable_value_and_suffix(
     )))
 }
 
+/// This is a function that converts a specific string format into a timestamp.
+/// It allows converting a time string of
+/// "(week abbreviation) (date), (year) (time)" to a Unix timestamp.
+/// such as: "jan 01, 2025 00:00:01" -> 1735689601000
+/// When (time) is not provided, it will be automatically filled in as 00:00:00
+/// such as: "jan 01, 2025" = "jan 01, 2025 00:00:00" -> 1735689600000
+fn parse_date_str_to_timestamps(date_str: &str) -> Option<i64> {
+    let regex_pattern = r"^(?:(\w{3} \d{2}))?(?:, (\d{4}))?(?: (\d{2}:\d{2}:\d{2}))?$";
```

Comment:
please use this kind of patterns:
make it easier to understand 

    let regex_pattern = r#"^(?P<month_day>\w{3} \d{2})?(?:, (?P<year>\d{4}))?(?: (?P<time>\d{2}:\d{2}:\d{2}))?$"#;

Reply:
Okay, this way of writing looks clearer! I will change it.